Formula/tree.rb

```diff
@@ -1,8 +1,8 @@
 class Tree < Formula
   desc "Display directories as trees (with optional color/HTML output)"
   homepage "http://mama.indstate.edu/users/ice/tree/"
-  url "https://deb.debian.org/debian/pool/main/t/tree/tree_1.8.0.orig.tar.gz"
-  sha256 "715d5d4b434321ce74706d0dd067505bb60c5ea83b5f0b3655dae40aa6f9b7c2"
+  url "http://mama.indstate.edu/users/ice/tree/src/tree-2.0.0.tgz"
```

Comment:
Is there no https?

Reply:
No there isn't

Reply:
Mirror seems to be down.

[![Screenshot 2021-12-24 at 06 31 35](https://user-images.githubusercontent.com/707714/147316548-816e5953-7120-4172-a361-0a585f335111.jpg)](https://downforeveryoneorjustme.com/mama.indstate.edu)